core/src/main/java/com/datastax/oss/driver/internal/core/channel/DriverChannel.java

```diff
@@ -49,9 +49,9 @@
 @ThreadSafe
 public class DriverChannel {
 
-  static final AttributeKey<String> CLUSTER_NAME_KEY = AttributeKey.newInstance("cluster_name");
+  static final AttributeKey<String> CLUSTER_NAME_KEY = AttributeKey.valueOf("cluster_name");
   static final AttributeKey<Map<String, List<String>>> OPTIONS_KEY =
-      AttributeKey.newInstance("options");
+      AttributeKey.valueOf("options");
```

Comment:
There's a static constant pool of AttributeKeys, so using valueOf will return the same instance that was defined previously, even across channels: https://github.com/netty/netty/blob/065a3478e9b1f8f0ebd09cf06cf394ff71fc5597/common/src/main/java/io/netty/util/AttributeKey.java#L25

I can't think of how that would cause any problems. If someone runs Channel.get(OPTIONS_KEY), it'll return the value for that Channel, which should be correct and expected behavior.

Reply:
:+1: thanks for reviewing.  Yeah, makes sense, another possible enhancement was to namespace the keys so its less likely for them to collide with other code, but I think it's incredibly unlikely, and given `DriverChannel`s underlying netty channel is private, its unlikely that it would be an issue.

Reply:
I think this might impact users who have separate classloaders for multiple driver instances, but a shared Netty instance, since each driver's static initializers will run in its own classloader, but the Netty constant pool will be shared. But this updated behavior seems more lenient than the existing behavior, so it still seems to me like it won't cause problems.

Reply:
Agreed all around... I'm having trouble seeing a reason why we _wouldn't_ want to do this.